src/main/kotlin/no/nav/familie/ef/iverksett/brukernotifikasjon/BrukernotifikasjonKafkaProducer.kt

```diff
@@ -45,6 +51,50 @@ class BrukernotifikasjonKafkaProducer(
         }
     }
 
+    fun sendBeskjedTilBrukerMedKafkaBuilder(
+        personIdent: String,
+        iverksettOvergangsstønad: IverksettOvergangsstønad,
+        behandlingId: UUID,
+    ) {
+        val generertVarselId = behandlingId.toString()
+
+        val opprettVarsel =
+            VarselActionBuilder.opprett {
+                type = Varseltype.Beskjed
+                varselId = generertVarselId
+                sensitivitet = Sensitivitet.High
+                ident = personIdent
+                aktivFremTil = null
+
+                tekst =
+                    Tekst(
+                        spraakkode = "nb",
+                        tekst = lagMelding(iverksettOvergangsstønad),
+                        default = true,
+                    )
+
+                produsent =
+                    Produsent(
+                        cluster = APPLICATION_CLUSTER,
```

Comment:
```
 @Value("\${NAIS_APP_NAME}")
    val applicationName: String,
    @Value("\${NAIS_NAMESPACE}")
    val namespace: String,
    @Value("\${NAIS_CLUSTER_NAME}")
    val cluster: String,
```

Disse kommer fra nais-plattformen ved oppstart.
Må legge inn "custom" i application-local.yml for å kunne kjøre lokalt